front_end/webserver.py

```diff
@@ -535,32 +544,35 @@ def post(self, course, assignment, problem):
                 if content.has_duplicate_title(content.get_problems(course, assignment), problem_basics["id"], title):
                     result = "Error: A problem with that title already exists in this assignment."
                 else:
-                    if (data_url == "" and data_file_name != "") or (data_url != "" and data_file_name == ""):
-                        result = "Error: If a data URL or file name is specified, both must be specified."
+                    if len(title) > 50:
```

Comment:
Please change this to "Error: The title cannot exceed..." That way it uses valid grammar and is consistent with the error message above.